numalogic/connectors/rds/db/__init__.py

```diff
@@ -0,0 +1,18 @@
+import importlib
+import os
+
+# value is an array of classes
+CLASS_TYPE = None
+
+"""This code snippet is a loop that iterates over the files in the directory of the current
+file. It checks if each file ends with "Fetcher.py" and if so, extracts the class name by
+removing the ".py" extension. It then imports the module corresponding to the class and
+retrieves the class object using the class name. Finally, it assigns the class object to the
+CLASS_TYPE variable. This code snippet is used to dynamically import and assign class objects
+based on file names."""
+
+for file_name in os.listdir(os.path.dirname(__file__)):
```

Comment:
let's use a factory method using a param approach. You can look into ModelFactories in the code.